app-modules/meeting-center/src/Managers/GoogleCalendarManager.php

```diff
@@ -166,6 +168,18 @@ public function syncEvents(Calendar $calendar, ?DateTime $start = null, ?DateTim
         $events
             ->each(
                 function (Event $event) use ($calendar) {
+                    // TODO EventTransparencyFeature: When removing the feature flag, replace lines 183-195 with:
+                    // $data = [
+                    //     'title' => $event->summary,
+                    //     'description' => $event->description,
+                    //     'starts_at' => $event->start->dateTime,
+                    //     'ends_at' => $event->end->dateTime,
+                    //     'attendees' => collect($event->getAttendees())
+                    //         ->map(fn (EventAttendee $attendee) => $attendee->getEmail())
+                    //         ->prepend($calendar->provider_email),
+                    //     'transparency' => EventTransparency::fromGoogleTransparency($event->getTransparency()),
+                    // ];
```

Comment:
Just for the future @payal-canyon, I wouldn't make feature flag translation this way.

I would do something like have array destructuring for `transparency` in the main data section. That way the only thing someone has to do is make it normal again.

Referencing line numbers and commenting whole chunks is a little hard to parse and also someone can make a change in this file before the FF is removed, changing the line numbers you specified.